elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java

```diff
@@ -697,6 +698,30 @@ public interface ElytronSubsystemMessages extends BasicLogger {
     @Message(id = 1211, value = "Unable to load the credential store.")
     OperationFailedException unableToLoadCredentialStore(@Cause Throwable cause);
 
+    @Message(id = 1212, value = "KeyStore does not contain a PrivateKey for KeyStore: [%s] and alias: [%s].")
+    StartException missingPrivateKey(String keyStore, String alias);
+
+    @Message(id = 1213, value = "KeyStore does not contain a PublicKey for KeyStore: [%s] and alias: [%s].")
+    StartException missingPublicKey(String keyStore, String alias);
+
+    @Message(id = 1214, value = "Unable to verify the integrity of the filesystem realm: %s")
+    OperationFailedException unableToVerifyIntegrity(@Cause Exception cause, String causeMessage);
+
+    @Message(id = 1215, value = "Filesystem is not configured with keypair")
```

Comment:
Maybe this could be reworded to something like: "Filesystem realm is missing key pair configuration, integrity checking is not enabled"

Reply:
Is this a duplicate of message 1219?

Reply:
> Is this a duplicate of message 1219?

Yes it is, I'll remove 1219

Reply:
> Maybe this could be reworded to something like: "Filesystem realm is missing key pair configuration, integrity checking is not enabled"

Fixed